staging/src/k8s.io/apiextensions-apiserver/third_party/forked/celopenapi/model/schemas.go

```diff
@@ -78,27 +104,26 @@ func SchemaDeclType(s *spec.Schema) *DeclType {
 				return UintType
 			}
 		}
-		return declType
 	}
-	return NewObjectTypeRef("*error*")
+	return declType
 }
 
 var (
 	// SchemaDef defines an Open API Schema definition in terms of an Open API Schema.
-	schemaDef *spec.Schema
+	schemaDef *schema.Structural
 
 	// AnySchema indicates that the value may be of any type.
-	AnySchema *spec.Schema
+	AnySchema *schema.Structural
 
 	// EnvSchema defines the schema for CEL environments referenced within Policy Templates.
-	envSchema *spec.Schema
+	envSchema *schema.Structural
```

Comment:
what is a policy template?

Reply:
It's the library we forked to get this CEL/OpenAPI type integration code: https://github.com/google/cel-policy-templates-go

We haven't 100% stripped out everything do don't need.

Reply:
I've done a pass and removed a lot more code from this directory.